strutil/strutil.go

```diff
@@ -203,3 +203,19 @@ func ElliptLeft(str string, n int) string {
 
 	return "…" + string(rstr[len(rstr)-n+1:])
 }
+
+// ListsSame verifies that two lists of strings have strictly the same strings,
+// in the same order.
+func ListsSame(l1, l2 []string) bool {
+	if len(l1) != len(l2) {
```

Comment:
Should we also consider that a nil slice is only equal to a nil slice?

Reply:
IMHO a nil slice is the same as `[]string{}` since they both are length 0 and have no elements to actually compare.